src/sidebar/components/Thread.js

```diff
@@ -129,4 +141,31 @@ function Thread({ thread, threadsService }) {
   );
 }
 
+/**
+ * Wrapper around AnnotationHeader
+ * @param {object} props
+ *   @param {Thread['annotation']} props.annotation
+ *   @param {number} props.replyCount
+ *   @param {boolean} props.threadIsCollapsed
+ */
+function AnnotationHeaderWrapper({ annotation, ...restProps }) {
```

Comment:
This is the wrapper component I created to emulate the same behaviour than when `AnnotationHeader` is called through `Annotation` component.